cloudigrade/api/error_codes.py

```diff
@@ -30,12 +29,13 @@ def get_message(self):
         """Get the external message for an error."""
         return self.message % {"error_code": self.code}
 
-    def notify(self, account_number, application_id, error_message=None):
+    def notify(self, application_id, error_message=None):
         """Tell sources an application is not available because of error."""
+        from cloudigrade.api.tasks import notify_application_availability_task
```

Comment:
the `cloudigrade.` root on this and similar import paths (like in `models.py` and the various tests) should not be necessary, and it may actually cause problems because the `PYTHONPATH` for our service should already be at that location.

Reply:
👍  much nicer and shorter.  Thank goodness for PyCharm find and replace 😄